<launch>

    <!-- Launch recorder node -->
    <node pkg="prog_rosbag" type="prog_record" name="recorder_node" output="screen" respawn="true">
        <param name="topic_name_file"  value="$(find mowbot)/config/rosbagtopics.config"/>
        <param name="data_directory"   value="/home/shaun/Desktop"/>
        <param name="file_prefix"      value="turtle_test_bag"/>
        <param name="start_topic"      value="/record/start"/>
        <param name="stop_topic"       value="/record/stop"/>
    </node>

    <node pkg="turtlesim" type="turtlesim_node" name="turtle_sim_node" output="screen"/>
</launch>
